source/buildspec.yml

```diff
@@ -0,0 +1,35 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      ruby: 2.7
+    commands:
+      - pip3 install --upgrade aws-sam-cli 
+
+  build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR
+      - GITSHA=$(cat gitsha.txt)
+      - echo $GITSHA
```

Comment:
do we need to save the `gitsha.txt` to the workspace between jobs?

Reply:
No.  This is just a way to include the sha in the build artifacts to be used during the deploy.